src/remote/action.go

```diff
@@ -165,7 +165,7 @@ func (c *Client) buildTestCommand(target *core.BuildTarget) (*pb.Command, error)
 			},
 		},
 		Arguments: []string{
-			c.bashPath, "--noprofile", "--norc", "-u", "-o", "pipefail", "-c", commandPrefix + cmd,
+			c.bashPath, "--noprofile", "--norc", "-u", "-o", "pipefail", "-e", "-c", commandPrefix + cmd,
```

Comment:
I guess we should centralise this set of flags somewhere...

Reply:
I guess so, maybe BASH should be defined as tool as well ?

Reply:
I suppose we could have a `shell` config option (though setting this to anything other than bash would mean you can't use any of our built in rules). 

I can't quite see how we'd define bash as a tool because defining something as a tool (currently) means setting an bash env variable and we can't do that without bash.